components/nexus-common/src/test/java/org/sonatype/nexus/common/validation/BeanValidationTrial.groovy

```diff
@@ -0,0 +1,117 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.common.validation
+
+import org.junit.Before
+import org.junit.Test
+import org.sonatype.sisu.litmus.testsupport.TestSupport
+
+import javax.validation.Valid
+import javax.validation.Validation
+import javax.validation.ValidatorFactory
+import javax.validation.constraints.NotNull
+
+/**
+ * Bean validation trials.
+ */
+class BeanValidationTrial
```

Comment:
what's the purpose of the 'trial's in this module? They don't run as part of the build, and seem to favor println over assertions